src/pydvl/utils/utility.py

```diff
@@ -91,20 +106,19 @@ def __init__(
         show_warnings: bool = False,
         enable_cache: bool = False,
         cache_options: Optional[MemcachedConfig] = None,
+        clone_before_fit: bool = True,
     ):
-        self.model = model
+        self.model = self._clone_model(model)
```

Comment:
We should document this too: if a clone is expensive, the user must know this will happen

Reply:
You're right. Sorry for not doing it.